platform-includes/configuration/auto-session-tracking/dart.flutter.mdx

```diff
@@ -16,16 +16,22 @@ Requires SDK version `9.0.0` or higher.
 
 </Alert>
 
-On web you're required to use the `SentryNavigatorObserver` since sessions are tied to navigation events.
+On web you're required to use the `SentryNavigatorObserver` since sessions are tied to navigation events and your navigation routes to have names, for example using `RouteSettings`.
```

Comment:
nit: this sentence structure sounds slightly off, but that could be subjective
```suggestion
On web it's required to use the `SentryNavigatorObserver` since sessions are tied to navigation events, and for your navigation routes to have names, for example using `RouteSettings`.
```